UPDATING.md

```diff
@@ -3,6 +3,16 @@
 This document will hold all information on how to update between major versions.
 
 
+## Update from `v1.6.2` to `v1.6.4`
```

Comment:
@cytopia Are you sure about version number 1.6.4 ;-) ?

Reply:
Good catch ;-)